src/pages/detailedpokemon/[id].tsx

```diff
@@ -76,6 +115,22 @@ export default function DetailedPokemon() {
         return total + current.base_stat;
     }, 0);
 
+    const engName = pokemonSpecies.names.find(
+        (pokename) => pokename.language.name === "en"
+    );
+    if (engName === undefined) {
+        return "Pokemon Name is missing.";
+    }
+
+    const evolutionNames = evolutionSpecies.map((evolutionSpecies) => {
+        return evolutionSpecies.names.find(
+            (evolutionName) => evolutionName.language.name === "en"
+        );
+    });
+    if (evolutionNames === undefined) {
```

Comment:
This condition will never be true. Also, not rendering the whole page would not be a good behavior for this anyway.